### ISSUE TITLE
Fiat deposits not working

### ISSUE THREAD

**Issue by dan-turner** [openware/peatio#276]
Appears to be due to a problem with CSRF token:

```
Started POST "/fund_sources" for 35.227.218.32 at 2018-01-11 05:45:43 +0000
Processing by Private::FundSourcesController#create as HTML
  Parameters: {"uid"=>"Dan Turner", "extra"=>"cbc", "currency"=>"usd", "fund_source"=>{"currency"=>"usd", "extra"=>"cbc", "uid"=>"Dan Turner"}}
Can't verify CSRF token authenticity
Completed 422 Unprocessable Entity in 2ms (ActiveRecord: 0.0ms)

ActionController::InvalidAuthenticityToken (ActionController::InvalidAuthenticityToken):
  lib/middleware/security.rb:11:in `call'
```

<img width="1094" alt="screenshot 2018-01-11 16 49 35" src="https://user-images.githubusercontent.com/814113/34810442-719aa4d4-f6ef-11e7-9f46-37c5c2683c34.png">

**Comment by yivo** [comment]
@dan-turner Thanks for your report.

**Comment by yivo** [comment]
Closed in #288.
